uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/controllers/request-ctrl.js

```diff
@@ -32,8 +32,9 @@ define([
       this.absencePeriods = [];
       this.absenceTypes = [];
       this.calendar = {};
-      this.contact = {};
+      this.contact = null;
```

Comment:
I think this variable is to store "manager's selected staff contact"? Then please give a more appropriate name like `contactSelectedByManager`